news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,103 @@
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="soonTM" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Touch screen's rise and fall (2015-2017)
+
+The earliest mentions of using touch screens as an input device came as soon as the game was released, again due to osu!'s roots as an *Osu! Tatakae! Ouendan* clone. Back then touch screens on PCs were not as prevalent, and most touch screen devices were "Tablet PCs", which were laptops that had a keyboard that can be rotated, folded, or outright detached, along with a touchscreen.
+
+It would take many years for the community to see the true potential of using touch screens in osu!, but starting in 2015 we started seeing glimpses, and eventually the peak of the play style in 2017 with the two players noted below.
+
+### Kuvster (2015)
+
+In 2015, [Kuvster](https://osu.ppy.sh/users/4896624) had a unique opportunity — snag the pp record under a very specific set of circumstances. Back in the day "TAG4" maps were [approved](/wiki/Beatmap/Category#approved), hence you were able to get pp on them. They weren't playable for a regular player, but if you had a touch screen and were apt with it, you were eligible to grab the pp record, which is what they did:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/jcHiuNcwsiM" frameborder="0" allowfullscreen></iframe>
+</div>
+
+The 577pp record was short-lived though, as after this all TAG4 maps were exempt from giving pp, and in 2019 TAG4 maps were moved to the [Loved category](/wiki/Beatmap/Category#loved). This however is not the event what "killed" touch screen, its demise would come only 2 years later.
+
+### Freedomdiver (2017)
+
+[Freedomdiver](https://osu.ppy.sh/users/8482062) joined the osu! community in May 2016, and was your average player, up until the moment they realised they can play osu! with the touch screen on their laptop.
+
+They quickly surged through the ranks after realizing that gaining pp by playing jump maps with a touch screen was easier, but not without garnering negative attention from the community. The community had turned against them, using them as a prime example of why touch screen is unfair, but it wouldn't be until November 2017 when freedomdiver decided to push touch screen to its limits, siding with the community and using actions instead of words to call for a change.
```

Comment:
```suggestion
They quickly surged through the ranks after realizing that gaining pp by playing jump maps with a touch screen was easier, but not without garnering negative attention from the community. The community had turned against them, using them as a prime example of why touch screen is unfair. It wouldn't be until November 2017 when freedomdiver decided to push touch screen to its limits, siding with the community and using actions instead of words to call for a change.
```

Reply:
applied venix's feedback that addresses this as well